pkgs/development/python-modules/cryptography/default.nix

```diff
@@ -35,15 +34,13 @@ buildPythonPackage rec {
     cffi
   ];
 
-  buildInputs = [ openssl ]
+  buildInputs = [ openssl setuptools-rust ]
```

Comment:
if this isn't meant to be used at runtime, it should probably go into nativeBuildInputs

Reply:
I was actually never sure about this for `buildPythonPackage`. Is our manual outdated in this regard?

Because e.g. https://nixos.org/manual/nixpkgs/unstable/#python ("15.20.1.2.2. Handling dependencies") states:
> According to the manual, buildPythonPackage uses the arguments buildInputs and propagatedBuildInputs to specify dependencies. If something is exclusively a build-time dependency, then the dependency should be included in buildInputs, but if it is (also) a runtime dependency, then it should be added to propagatedBuildInputs. Test dependencies are considered build-time dependencies and passed to checkInputs.

(I'm also not sure if "According to the manual" in the manual is an accidental self-reference :D)

Reply:
@jonringer @FRidh should `setuptools-rust` be part of `nativeBuildInputs` or `buildInputs` now?

Reply:
nativeBuildInputs